clients/admin-ui/src/features/user-management/UserManagementRow.tsx

```diff
@@ -24,46 +25,67 @@ interface UserManagementRowProps {
 
 const UserManagementRow: React.FC<UserManagementRowProps> = ({ user }) => {
   const router = useRouter();
+  const modal = useDisclosure();
```

Comment:
Super nit here 😬 Could this be renamed to `deleteModal` instead? It'll make it obvious what modal it goes too and if we add more modals in the future this one won't accidentally get reused.

Reply:
heh you got it!